test/benchmarks/README.md

```diff
@@ -0,0 +1,51 @@
+# Benchmarks
```

Comment:
This is not too far off something that already happens in an e2e test. I'd like to see a new github action to test this.

That framework has the ability to time tests (and already does to show us slow tests), and this is somewhat akin to the CLI test suite in that you're wanting to invoke command line actions (albeit not the argocli) and monitor the result.

You can try all this out in github in your own fork and see whether a CI test is feasible. If you take the necessary github action file and add [`workflow_dispatch`](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow) as a trigger event you can then also manually trigger it if you want to try the same thing over and over (as might be prudent here).

Reply:
I see. I added it here as @agilgur5 suggested, but I didn't have time to automate tests. Do you think it's better to remove them or keep them here?

Reply:
I was hoping that you'd consider automating the tests.

Reply:
I mentioned automating a [simple "performance regression test" too](https://cloud-native.slack.com/archives/C0510EUH90V/p1727332000712739?thread_ts=1727281453.125669&cid=C0510EUH90V), but a full benchmark may not be truly possible in CI. See also discussion in #13767 

This may be difficult to do within the confines of `go test`, but I suppose the script could be a file as opposed to an example. #13767 would be good to document on this new page too